src/tasks/AWSShellScript/task.json

```diff
@@ -111,7 +112,7 @@
         }
     ],
     "execution": {
-        "Node": {
+        "Node10": {
```

Comment:
So could tasks hypothetically be executable on multiple node versions?

Reply:
Yes, but I don't believe there's a deterministic selector to force Node10 execution (it might come down to the Azure Pipelines Agent's discrimination). Execution handler definitions can be found here: https://github.com/Microsoft/azure-pipelines-task-lib/blob/master/tasks.schema.json